homeassistant/components/zha/core/channels/closures.py

```diff
@@ -23,6 +23,24 @@ async def async_update(self):
                 f"{self.unique_id}_{SIGNAL_ATTR_UPDATED}", 0, "lock_state", result
             )
 
+    @callback
+    def cluster_command(self, tsn, command_id, args):
+        """Handle a cluster command received on this cluster."""
+
+        if self._cluster.client_commands is None:
+            return
+
+        command_name = self._cluster.client_commands.get(command_id, [command_id])[0]
+        if command_name == "operation_event_notification":
+            self.zha_send_event(
+                command_name,
+                {
+                    "source": args[0].name,
```

Comment:
@Adminiuga is there a better way to parse the args out? Or is there a better way to do this?

Reply:
not really, cause args would change depending on the command_id